code/modules/mob/living/simple_animal/hostile/netherworld.dm

```diff
@@ -70,28 +80,43 @@
 	icon_state = "nether"
 	max_integrity = 50
 	spawn_time = 600 //1 minute
-	max_mobs = 15
+	max_mobs = 10
 	icon = 'icons/mob/nest.dmi'
 	spawn_text = "crawls through"
 	mob_types = list(/mob/living/simple_animal/hostile/netherworld/migo, /mob/living/simple_animal/hostile/netherworld, /mob/living/simple_animal/hostile/netherworld/blankbody)
-	faction = list("nether")
+	faction = list("nether", "mining", "cult")
+	var/blankckey = null
 
 /obj/structure/spawner/nether/Initialize(mapload)
 	.=..()
 	START_PROCESSING(SSprocessing, src)
 
 /obj/structure/spawner/nether/examine(mob/user)
 	..()
-	if(isskeleton(user) || iszombie(user))
-		to_chat(user, "A direct link to another dimension full of creatures very happy to see you. <span class='nicegreen'>You can see your house from here!</span>")
+	if(iscultist(user))
```

Comment:
Why did you remove the fluff text for skeleton and zombie, and replace it for cultists only.